tools/configure/configure.ml

```diff
@@ -209,7 +204,7 @@ let check_for_zarith prefs =
 let check_sphinx_deps () =
   ignore (run (which "python3") ["doc/tools/coqrst/checkdeps.py"])
 
-let check_doc () =
+let _check_doc () =
```

Comment:
Why keep?

Reply:
As a reminder to think where we should perform this check.

Reply:
Ok, fixed and fixed stale docs about docs.